config/default.js

```diff
@@ -4,6 +4,9 @@ module.exports = {
   hash: {
     salt: 8
   },
+  session: {
+    secret: 'l1ClK9xjAp0LzW0hpByT'
```

Comment:
![Slika](https://media.giphy.com/media/Pn1gZzAY38kbm/giphy.gif)

Reply:
To be clear, it's even ok to have dev env secret up, but it then must be in `config/development.js`.